src/components/HealthChecksSummary/index.tsx

```diff
@@ -28,12 +28,12 @@ export function HealthChecksSummary({
       warning: 0
     };
 
-    checks.forEach((check) => {
-      const status = check.status;
-      summary.healthy += status === "healthy" ? 1 : 0;
-      summary.warning += status === "warning" ? 1 : 0;
-      summary.unhealthy += status === "unhealthy" ? 1 : 0;
-    });
+    // checks.forEach((check) => {
```

Comment:
?

Reply:
checks are not an array, but an object, I am following up with Yash to understand what this means